build.gradle

```diff
@@ -19,6 +19,7 @@ buildscript {
         classpath 'com.google.firebase:firebase-crashlytics-gradle:2.8.1'
         classpath("io.gitlab.arturbosch.detekt:detekt-gradle-plugin:1.20.0-RC1")
         classpath("com.vanniktech:gradle-android-junit-jacoco-plugin:0.16.0")
+        classpath 'org.jetbrains.kotlin:kotlin-gradle-plugin:1.7.20'
```

Comment:
Temporary changes which are required for the Testing sample to run. would revert